src/Renci.SshNet/ScpClient.NET.cs

```diff
@@ -15,6 +16,14 @@ public partial class ScpClient
         private static readonly Regex TimestampRe = new Regex(@"T(?<mtime>\d+) 0 (?<atime>\d+) 0");
 
         /// <summary>
+        /// Transferring UNIX files to Windows may require special treatment.
+        /// Enabling this will remove the following special characters from filenames when copying whole directories:
+        /// " &lt; &gt; | : * ? \ /
+        /// </summary>
+        public bool UnixToWindows { get; set; }
```

Comment:
Do we need this extra property, or can we always enable this (or only when running on Windows)?
In that case, we should add a remark for this to ```ScpClient.Download(string, DirectoryInfo)```.